### PR TITLE
kotlin Migration UtilsTest

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Purpose / Description
Kotlin migration of UtilsTest
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/libanki/UtilsTest.kt [hunk 1]
```diff
@@ -14,142 +14,128 @@
...
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.File
+import java.io.IOException
+import java.lang.Exception
+import java.util.*
+import kotlin.Throws
+
+@RunWith(AndroidJUnit4::class)
+class UtilsTest {
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/libanki/UtilsTest.kt:34]: `KotlinCleanup("IDE Lint")` - presumably quite a lot of IDE suggestions here

**Review by david-allison — CHANGES_REQUESTED** [review]
looks great!

**Review by david-allison — APPROVED** [review]
awesome!

**Review by mikehardy — APPROVED** [review]
Nice!

**Pull request merged**